core/src/test/java/kafka/testkit/KafkaClusterTestKitTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kafka.testkit;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class KafkaClusterTestKitTest {
+    @Test
+    public void testCreateClusterWithNoDisksThrows() {
+        try (KafkaClusterTestKit cluster = new KafkaClusterTestKit.Builder(
```

Comment:
We can use `assertThrows` to rewrite it. For example:
```java
        Exception e = assertThrows(Exception.class, () -> new KafkaClusterTestKit.Builder(
                new TestKitNodes.Builder().
                        setBrokerNodes(1, 0).
                        setNumControllerNodes(1).build()));
        assertEquals(RuntimeException.class, e.getClass());
        assertEquals("Invalid value for disksPerBroker", e.getMessage());
```

Reply:
👍 Using `assertThrowsExactly` instead to avoid the equality check on `RuntimeException.class`